src/testUtils/mocks/dataSources.ts

```diff
@@ -892,6 +896,20 @@ export function setContextAccountBalance(balance: AccountBalance): void {
   mockInstanceContainer.contextInstance.accountBalance.returns(balance);
 }
 
+/**
+ * @hidden
+ * Sets an address in the mocked Context
+ *
+ * @param address - new address account
+ */
+export function setContextAccountAddress(address: string): void {
```

Comment:
You should make the result of `getAccounts` be configurable as part of `ContextOptions` instead of doing this